includes/wikia/GlobalFunctions.php

```diff
@@ -1613,3 +1613,15 @@ function wfGetUniqueArrayCI( array $arr ) {
 	$unique = array_intersect_key( $arr, array_unique( $lower ) );
 	return array_filter( $unique );
 }
+
+/**
+ * Like pathinfo but with support for multibyte - copied from http://php.net/manual/en/function.pathinfo.php#107461
+ */
+function mb_pathinfo($filepath) {
```

Comment:
I know this is copied from elsewhere, but I still think it should use our coding conventions (spacey, always use braces on blocks, etc.).